src/gov/nist/javax/sip/parser/AddressParametersParser.java

```diff
@@ -64,7 +66,15 @@ protected void parse(AddressParametersHeader addressParametersHeader)
                  super.parseNameValueList(addressParametersHeader);
 
 
-            }  else super.parse(addressParametersHeader);
+            }  else {
+            	lexer.SPorHT();            	
+            	if (this.lexer.lookAhead(0) == ';') {
+            		if (this.allowParameters == false) {
+                		throw new ParseException(this.lexer.getBuffer() +  "is not valid. This Header Field cannot has Parameters" , this.lexer.getPtr());
```

Comment:
error text its a bit strange in terms of english ("cannot has" is not correct, "can't have" in any case..). something like this would be better though ...
"is not valid. This header doesn't allow parameters"